src/connectors/configure/connectConfigure.js

```diff
@@ -1,18 +1,21 @@
-import isFunction from 'lodash/isFunction';
+import noop from 'lodash/noop';
```

Comment:
another time we should either converge on using the lodash or a local noop, it's inconsistent now :)